src/dune_rules/dune_package.ml

```diff
@@ -297,8 +297,7 @@ let decode ~lang ~dir =
   let+ name = field "name" Package.Name.decode
   and+ version = field_o "version" string
   and+ sections =
-    field ~default:[] "sections"
-      (repeat (pair (located Section.decode) Dpath.decode))
+    field ~default:[] "sections" (repeat (pair (located Section.decode) string))
```

Comment:
A little cleaner I think is to add an `External` sub-module to `Dpath` and use `Dpath.External.decode` directly.